test/Tabs.test.js

```diff
@@ -136,6 +136,7 @@ describe('tabs behaviour', () => {
 			fixtures.reset();
 			fixtures.insertSimple();
 			withUpdateUrl && tabsEl.setAttribute('data-o-tabs-update-url', '');
+			tabsEl.setAttribute('data-o-tabs-disablefocus', 'false');
```

Comment:
@AlbertoElias just looked into this a bit further, and yes it is the default in the script but in https://github.com/Financial-Times/o-tabs/blob/master/test/helpers/fixtures.js#L24 you've set it to be true for another test. So this should set it back to false for the hash tests.